contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -98,6 +193,35 @@ public void run() throws Exception {
         JobConf conf = new JobConf();
         Path dataPath = new Path(dataDir);
         String modifiedDataDir = dataPath.makeQualified(FileSystem.get(conf)).toString();
+        String cdnURL = pickCDN();
+
+        if (!cdnURL.isEmpty()) {
+            info("Using CDN: " + cdnURL);
+            info("####################################");
+            info("              Distcp");
+            info("####################################");
+            try {
+                String username = props.getString("env.USER", "unknownClient");  // TODO: Is this attribute always existing?
```

Comment:
I think it possible to have conflict in that case. It doesn't hurt to have something extra. but I'll add storename to the path.